src/components/numpad/Numpad.js

```diff
@@ -10,18 +11,22 @@ import { Text } from '../text';
 const KeyColor = colors.alpha(colors.blueGreyDark, 0.8);
 const defaultTransform = { transform: [{ scale: 0.5 }] };
 
-const KeyboardButton = ({ children, ...props }) => (
-  <ButtonPressAnimation
-    {...props}
-    duration={35}
-    scaleTo={1.6}
-    transformOrigin="keyboard"
-  >
-    <Centered height={64} style={defaultTransform} width={80}>
-      {children}
-    </Centered>
-  </ButtonPressAnimation>
-);
+const KeyboardButton = ({ children, ...props }) => {
+  const { isTinyPhone } = useDimensions();
```

Comment:
All this calc can be done outside of the component so it doesn't run on every render.

Reply:
NVM, totally missed that useDimensions was a hook 😅